assets/localisation/alice.csv

```diff
@@ -1,38 +1,60 @@
-alice_save_exit;Save and Exit;;;Salir y Guardar;;;;;;;;;x
-alice_save;Save;;;Guardar;;;;;;;;;x
-alice_exit;Exit;;;Salir;;;;;;;;;x
+language_en-US;English (US)
+language_fr-FR;French (France)
```

Comment:
See previous comment about displaying language names. In any case, displaying the language name should bypass the localization system entirely because it should always display with its native name, not translated.

Reply:
We are going to eventually show a list of supported languages, hence the multiple definitions